manta-crypto/src/arkworks/mod.rs

```diff
@@ -34,6 +34,7 @@ pub use ark_ed_on_bls12_381 as ed_on_bls12_381;
 pub use ark_ed_on_bn254 as ed_on_bn254;
 
 pub mod algebra;
+pub mod codec;
```

Comment:
Like I mentioned before, we can use `serialize` here instead and do an extension of `ark-serialize` in that module.